hexa/pipelines/schema/mutations.py

```diff
@@ -43,11 +44,35 @@ def resolve_create_pipeline(_, info, **kwargs):
         }
 
     try:
-        pipeline = Pipeline.objects.create(
-            code=input["code"],
-            name=input.get("name"),
-            workspace=workspace,
-        )
+        notebook = input.get("notebookPath")
```

Comment:
It's better to clearly state what is in your variable
```suggestion
        notebook_path = input.get("notebookPath")
```